.zuul.d/project.yaml

```diff
@@ -3,6 +3,7 @@
     check:
       jobs:
         # - ansible-builder-tox-integration
+        - ansible-tox-docs
```

Comment:
you should add this to gate below also.

Reply:
yup, just wanted it working in check first. done